src/components/formio/textfield/textfield.component.tsx

```diff
@@ -1,39 +1,38 @@
-import {CharCount, Component, Description, Errors, Label} from '@components';
 import {IComponentProps} from '@types';
 import clsx from 'clsx';
 import {ComponentSchema} from 'formiojs';
 import React, {useState} from 'react';
 
+import {CharCount, Component, Description, Errors, Label} from '../../utils';
+
 export interface ITextFieldComponent extends ComponentSchema {
   id: string;
   inputMask: string;
+  key: string;
+  label: string;
   mask: string;
+  showCharCount: true;
```

Comment:
```suggestion
  showCharCount?: boolean;
```
not a literal, nor is it guaranteed to be present unfortunately (luckily enough undefined/null is falsy)

Reply:
weird, dunno why it's true here